src/twisted/trial/test/test_tests.py

```diff
@@ -965,6 +965,28 @@ def test_multipleErrorsReported(self):
         self.assertEqual(error2.getErrorMessage(), "foo")
 
 
+    def test_cleanupsAreReentrant(self):
```

Comment:
Is "reentrant" the correct term here? It suggests both callbacks are on the call stack simultaneously, which is not the case.